### PR TITLE
SC: Rename main/service->parent/child, bridge account -> operator

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes
Service chain can be composed for multiple layer chains like below.
- main net <-> 1st service chain <-> 2nd service chain <-> ... <-> Nth service chain

For this multiple layer service chains, main/service chain words are not exact.
So this PR renamed some words like below.
- main/service->parent/child

And we renamed bridge account to operator which sign the handled transaction for bridge contract. This will be expected to help to understand well of bridge contract structure.
- bridge account -> operator

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: client/bridge_client.go [hunk 1]
```diff
@@ -216,28 +216,28 @@ func (ec *Client) BridgeListBridge(ctx context.Context) ([]*BridgeJournal, error
 	return result, err
 }
 
-// BridgeSubscribeBridge can enable for service chain bridge to subscribe the event of given service/main chain bridges.
+// BridgeSubscribeBridge can enable for service chain bridge to subscribe the event of given child/parent chain bridges.
```
Comment (jeonghonah) [inline client/bridge_client.go]:
```suggestion
// BridgeSubscribeBridge can enable for subbridge to subscribe the event of given child/parent chain bridges.
```

File: node/sc/main_event_handler.go [hunk 2]
```diff
@@ -73,11 +73,11 @@ func (mce *MainChainEventHandler) WriteLastIndexedBlockNumber(blockNum uint64) {
 	mce.mainbridge.chainDB.WriteLastIndexedBlockNumber(blockNum)
 }
 
-// ConvertServiceChainBlockHashToMainChainTxHash returns a transaction hash of a transaction which contains
+// ConvertChildChainBlockHashToParentChainTxHash returns a transaction hash of a transaction which contains
 // ChainHashes, with the key made with given service chain block hash.
```
Comment (jeonghonah) [inline node/sc/main_event_handler.go]:
```suggestion
// ChainHashes, with the key made with given child chain block hash.
```

File: node/sc/main_event_handler.go [hunk 3]
```diff
@@ -73,11 +73,11 @@ func (mce *MainChainEventHandler) WriteLastIndexedBlockNumber(blockNum uint64) {
...
 // Index is built when service chain indexing is enabled.
```
Comment (jeonghonah) [inline node/sc/main_event_handler.go]:
```suggestion
// Index is built when child chain indexing is enabled.
```

File: node/sc/vt_recovery_test.go [hunk 4]
```diff
@@ -364,7 +364,7 @@ func TestAlreadyStartedVTRecovery(t *testing.T) {
 	vtr.Stop()
 }
 
-// TestScenarioMainChainRecovery tests the value transfer recovery of the main chain to service chain value transfers.
+// TestScenarioMainChainRecovery tests the value transfer recovery of the parent chain to service chain value transfers.
```
Comment (jeonghonah) [inline node/sc/vt_recovery_test.go]:
```suggestion
// TestScenarioMainChainRecovery tests the value transfer recovery of the parent chain to child chain value transfers.
```

File: node/sc/vt_recovery_test.go [hunk 5]
```diff
@@ -719,11 +719,11 @@ func requestNFTTransfer(info *testInfo, bi *BridgeInfo) {
 	defer bi.account.UnLock()
 
 	opts := bi.account.GetTransactOpts()
-	// TODO-Klaytn need to separate service / main chain nftIndex.
+	// TODO-Klaytn need to separate service / parent chain nftIndex.
```
Comment (jeonghonah) [inline node/sc/vt_recovery_test.go]:
```suggestion
	// TODO-Klaytn need to separate child / parent chain nftIndex.
```

**Comment by KimKyungup** [comment]
@jeonghonah Thanks. :) I reflected.

File: node/sc/config.go [hunk 6]
```diff
@@ -36,7 +36,7 @@ import (
 )
 
 const (
-	datadirMainChainBridgeNodes = "mainchain-bridges.json" // Path within the datadir to the static node list
+	datadirMainChainBridgeNodes = "parentchain-bridges.json" // Path within the datadir to the static node list
```
Comment (jeonghonah) [inline node/sc/config.go]: How about `main-bridges.json`?

Comment (KimKyungup) [reply node/sc/config.go]: Good. :)

Comment (KimKyungup) [inline node/sc/config.go]:
```suggestion
	datadirMainChainBridgeNodes = "main-bridges.json" // Path within the datadir to the static node list
```

**Comment by KimKyungup** [comment]
https://github.com/ground-x/klaytn-node-tests/pull/90

**Comment by KimKyungup** [comment]
> @KimKyungup Could you also write about what made you these changes in the description of this PR? Or, you could create an issue describing problems that initiated this PR.

@smseo I will write some reason why this PR is needed.

**Pull request merged**